lib/ansible/executor/module_common.py

```diff
@@ -1357,9 +1357,9 @@ def get_action_args_with_defaults(action, args, defaults, templar, redirected_na
         'aws': ['amazon.aws', 'community.aws'],
         'azure': ['azure.azcollection'],
         'cpm': ['wti.remote'],
-        'docker': ['community.general'],
+        'docker': ['community.general', 'community.docker'],
```

Comment:
not a blocker to this PR, but this should be moved to config (really should not be a thing in core at all, but setlingling for moving into config).

Reply:
I was assuming that this is a placeholder anyway until the action groups feature is fully designed and implemented.

Reply:
I believe that is tentatively slated for 2.11